src/bun.js/node/types.zig

```diff
@@ -1672,7 +1672,7 @@ pub const Path = struct {
         }
     }
     fn isAbsoluteString(path: JSC.ZigString, windows: bool) bool {
-        if (!windows) return path.len > 0 and path.slice()[0] == '/';
+        if (!windows) return path.len > 0 and path.ptr[0] == '/';
```

Comment:
this is a segfault

we should never read from `ptr` in JSC.ZigString directly

i'll make it more clear in the code that we shoulnd't do that

Reply:
both are actually segfaults, i'll fix